src/components/NavBar/LogIn/LogIn.js

```diff
@@ -1,7 +1,31 @@
 import React, { Component } from 'react';
+import './LogIn.css';
 
 class LogIn extends Component {
 
+  componentDidMount() {
+    let username = localStorage.getItem('username');
+    if(username) {
+      document.getElementById('input-username-login').value = username;
+      document.getElementById('remember-username').checked = 'checked';
+    }
+  }
+
+  handleSubmit = (e) => {
+    e.preventDefault();
+    let username = document.getElementById('input-username-login').value;
+    let password = document.getElementById('input-password-login').value;
+    if(document.getElementById('remember-username').checked) {
```

Comment:
@MartyTheeMartian `const` rather than `let`, yes?

Reply:
@elimisteve I initially wrote those lines with `const` but I wasn't sure if we were going to reuse those variables after they've been encrypted and returned.